src/transformStateWithClones.js

```diff
@@ -6,8 +6,35 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  const temp = { ...state };
+  const transformedState = [];
+
+  for (const keys in actions) {
```

Comment:
Use `for...of` instead of `for...in` to iterate over the `actions` array. `for...in` is intended for iterating over object properties, not array elements.